src/components/About.tsx

```diff
@@ -0,0 +1,41 @@
+type AboutProps = {
+    photoUrl?: string; // optional for now
```

Comment:
Rather than make it optional for now, use something like https://placehold.co/600x400